tests/test_taxi.py

```diff
@@ -69,11 +67,11 @@ def test_car_is_register_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists("../.gitignore")
         assert file_exists
```

Comment:
The use of `assert` is not recommended in Django test cases. Instead, you should use `self.assertTrue(file_exists)` to ensure that the test runner properly recognizes and reports test failures.